packages/block-library/src/group/edit.native.js

```diff
@@ -0,0 +1,50 @@
+
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import {
+	InnerBlocks,
+	withColors,
+} from '@wordpress/block-editor';
+/**
+ * Internal dependencies
+ */
+import styles from './editor.scss';
+
+function GroupEdit( {
+	hasInnerBlocks,
+	isSelected,
+} ) {
+	return (
+		<>
+			{ ! isSelected && ! hasInnerBlocks ?
```

Comment:
It took me a while to parse that there was a ternary operator here, it looks like it would render both the View and the InnerBlocks. I don't have a specific preference, but I think this could be written in a more readable way. For instance:

```jsx
if ( ! isSelected && ! hasInnerBlocks ) {
	return (
		<View style={ styles.groupPlaceholder } />
	)
}

return (
	<InnerBlocks
		renderAppender={ isSelected && InnerBlocks.ButtonBlockAppender }
	/>
);
```